actors/util/adt/map.go

```diff
@@ -117,10 +117,18 @@ func (h *Map) ForEach(out vmr.CBORUnmarshaler, fn func(key string) error) error
 	})
 }
 
+type keyWrapper struct {
+	string
+}
+
+func (kw *keyWrapper) Key() string {
+	return kw.string
+}
+
```

Comment:
This change make usage of the `CollectKeys()` method more ergonomic as it allows for values to be immiadetly fetched by their keys without casting, e.g.:
```go
keys, err := txns.CollectKeys()
var txn multisig.MultiSigTransaction
found, err := txns.Get(keys[0], &txn)
```